lms/envs/common.py

```diff
@@ -1312,6 +1316,18 @@
         ],
         'output_filename': 'css/lms-style-course-vendor.css',
     },
+    'style-edx-footer': {
+        'source_filenames': [
+            'sass/footer-v3.css'
```

Comment:
Minor: It's good practice to add a trailing comma to lines like this so that others adding elements to this list in the future get a clean diff.